lbry/extras/daemon/daemon.py

```diff
@@ -3107,6 +3121,8 @@ def jsonrpc_channel_list(self, account_id=None, wallet_id=None, page=None, page_
                     f"Use --allow-duplicate-name flag to override."
                 )
 
+        file_path = await self._video_file_analyzer.verify_or_repair(validate_file, optimize_file, file_path)
```

Comment:
Should this not even get called if `validate_file` and `optimize_file` are both `False`?

Reply:
That case is handled at the top of verify_or_repair.

Reply:
The `await` carries a non-trivial penalty though (the function will stop executing and go back onto the schedule and will not pickup again until much later) and especially right at the beginning of the function. In the cases where both are false it doesn't make sense to suspend the execution of the function just to run an if statement inside of `verify_or_repair`.

for example, lbry.tv and spee.ch call publish a lot so it's important not to introduce unnecessary performance penalties.